components/MessageBroker.js

```diff
@@ -0,0 +1,21 @@
+const aws = require('aws-sdk');
```

Comment:
just putting this here - can we make this folder `lib` instead of `components` - that's a React convention. Same with the capitalisation - tends to be with components / classes.